crates/meroctl/src/cli/context/create.rs

```diff
@@ -12,13 +13,17 @@ use crate::config_file::ConfigFile;
 #[derive(Debug, Parser)]
 pub struct CreateCommand {
     /// The application ID to attach to the context
-    #[clap(long, short = 'a', exclusive = true)]
+    #[clap(long, short = 'a')]
     application_id: Option<calimero_primitives::application::ApplicationId>,
 
     /// Path to the application file to watch and install locally
-    #[clap(long, short = 'w', exclusive = true)]
+    #[clap(long, short = 'w')]
     watch: Option<Utf8PathBuf>,
+
     metadata: Option<Vec<u8>>,
```

Comment:
out of scope for this PR, just thinking about this.. maybe a `String` makes more sense, as vecs imply multiple specifications of the arg or alternatively, comma-separated

```console
$ meroctl context create --metadata 1 --metadata 2
$ meroctl context create --metadata 1,2,3
```

let's just make it a string, and before we send it to the node, make it a vec

Reply:
i will skip it in this PR as you mentioned it is out of scope. can u create a proper issue so we can make it right at once?

Reply:
https://github.com/calimero-network/core/issues/615